pkg-manager/plugin-commands-installation/src/update/getUpdateChoices.ts

```diff
@@ -48,8 +48,8 @@ export function getUpdateChoices (outdatedPkgsOfProjects: OutdatedPackage[], wor
   const finalChoices: ChoiceGroup = []
   for (const [depGroup, choiceRows] of Object.entries(groupPkgsByType)) {
     if (choiceRows.length === 0) continue
-
-    const rawChoices = choiceRows.map(choice => buildPkgChoice(choice, workspacesEnabled))
+    const rawChoices = choiceRows.filter(choice => !choice.latestManifest?.deprecated).map(choice => buildPkgChoice(choice, workspacesEnabled))
```

Comment:
but won't this also filter out those that have a new version that is deprecated? We should filter out those that have the same version

Reply:
Indeed, it has been updated.